src/cnd_settings.rs

```diff
@@ -63,11 +67,13 @@ impl Default for Network {
     }
 }
 
-impl Default for HttpSocket {
-    fn default() -> HttpSocket {
-        HttpSocket {
-            address: IpAddr::V4(Ipv4Addr::UNSPECIFIED),
-            port: 8080,
+impl Default for HttpApi {
+    fn default() -> HttpApi {
+        HttpApi {
+            socket: Socket {
```

Comment:
if you don't specify CORS settings it will default to no origins allowed for CORS, which I think is not what we want in `cca`.